tests/urlpatterns_reverse/tests.py

```diff
@@ -361,6 +361,27 @@ def test_resolver_reverse(self):
         self.assertEqual(resolver.reverse('named-url2', 'arg'), 'extra/arg/')
         self.assertEqual(resolver.reverse('named-url2', extra='arg'), 'extra/arg/')
 
+    def test_resolver_reverse_conflict(self):
+        """
+        Verifies that url names can indeed conflict: they need not be unique.
```

Comment:
@timgraham hey, so how would like for me to document this, because I feel like I do follow the pattern set forth in the guidelines. In fact, I match the approach used around me in other documentation.